shopify/CHANGELOG.md

```diff
@@ -4,4 +4,4 @@ WISMO app
 
 - Data pull for customers and orders
 - Actions: lookup_order_by_order_num, lookup_order_by_email
-- Auth: headers and oauth
+- Auth: headers and oauth
```

Comment:
Please add the changes made between v2 and v3. Please also add to the PR description